e2e-tests/cross-site/run

```diff
@@ -76,13 +87,25 @@ main() {
 	sleep 15
 	wait_cluster_consistency "$replica_cluster" 3 2
 
-	desc "Check replication source-> replica works on"
+	desc "Check replication works between source -> replica"
 	compare_mysql_cmd "select-2" "SELECT * from myApp.testSourceReplica;" "-h $replica_endpoint0 -uroot -proot_password"
 	compare_mysql_cmd "select-2" "SELECT * from myApp.testSourceReplica;" "-h $replica_endpoint1 -uroot -proot_password"
 	compare_mysql_cmd "select-2" "SELECT * from myApp.testSourceReplica;" "-h $replica_endpoint2 -uroot -proot_password"
 
+	run_backup "$replica_cluster" "backup-minio-replica"
+
 	desc "Switch clusters over"
 	kubectl_bin config set-context $(kubectl_bin config current-context) --namespace="$namespace"
+
+	desc "rebuild source cluster"
+	destination=$(kubectl_bin get -n "${replica_namespace}" pxc-backup backup-minio-replica -o jsonpath='{.status.destination}')
+	cat "${test_dir}/conf/restore-backup-minio.yml" \
```

Comment:
If you put the pipes ("|") at the end of the lines, you will not need to use the backslash ("\"). For example:
```
$sed -e "s~#cluster~${source_cluster}~" |
$sed -e "s~#destination~${destination}~" |
$sed -e "s~minio-service.#namespace~minio-service.${namespace}~" |
kubectl_bin apply -f -
```
but that's just a suggestion.